src/BucketTesting/CampaignConfigurationLoader.php

```diff
@@ -25,7 +25,7 @@ public function __construct(
 	 */
 	public function loadCampaignConfiguration( string ...$configFiles ): array {
 		$cacheKey = $this->getCacheKey( ...$configFiles );
-		if ( $cacheKey !== '' && $this->cache->has( $cacheKey ) ) {
+		if ( $cacheKey !== '' && $this->cache->has( $cacheKey ) && is_array( $this->cache->get( $cacheKey ) ) ) {
```

Comment:
I think this would work and be a little more readable:

```php
$cache = $this->cache->get( $cacheKey, [] );
if ( isset( $cache[ 'campaigns' ] ) ) {
	return $cache[ 'campaigns' ];
}
```

Reply:
@Abban I did the changes that you suggested but then the initial error came back:

  Line   src/BucketTesting/CampaignConfigurationLoader.php  
 ------ --------------------------------------------------- 
  30     Cannot access offset 'campaigns' on mixed.         
         🪪  offsetAccess.nonOffsetAccessible  
         
         
What should be the next step then?